tests/dialects/test_tsql.py

```diff
@@ -1497,27 +1497,30 @@ def test_datepart(self):
         )
         self.validate_identity(
             "DATEPART(YEAR, x)",
-            "FORMAT(CAST(x AS DATETIME2), 'yyyy')",
+            "DATEPART(YEAR, CAST(x AS DATETIME2))",
         )
         self.validate_identity(
             "DATEPART(HOUR, date_and_time)",
             "DATEPART(HOUR, CAST(date_and_time AS DATETIME2))",
         )
         self.validate_identity(
             "DATEPART(WEEKDAY, date_and_time)",
-            "DATEPART(DW, CAST(date_and_time AS DATETIME2))",
+            "DATEPART(WEEKDAY, CAST(date_and_time AS DATETIME2))",
+        )
+        self.validate_identity(
+            "DATEPART(tz, date_and_time)",
+            "DATEPART(TZOFFSET, CAST(date_and_time AS DATETIMEOFFSET))",
```

Comment:
Ditto re: casting to `DATETIMEOFFSET` here. Or are these used to facilitate transpilation?